includes/class-wc-dintero-hp-checkout.php

```diff
@@ -1741,8 +1741,8 @@ public function process_cart() {
 				}
 
 				$round_precision = min(array(wc_get_price_decimals(), 2));
-				$amount = $cart_item['line_tax'] + $cart_item['line_total'];
-				$tax = $cart_item['line_tax'];
+				$tax = round($cart_item['line_tax'], $round_precision);
+				$amount = $tax + round($cart_item['line_total'], $round_precision);
```

Comment:
Could maybe round after addition as well. It's possible to get a new inaccuracy when adding